recipe/migrations/openvino202410.yaml

```diff
@@ -0,0 +1,9 @@
+migrator_ts: 1712592149
+__migrator:
+  kind: version
+  migration_number: 1
+  bump_number: 1
+openvino:
+  - 2024.1.0
```

Comment:
```suggestion
```

if that is the case, we can delete this too. Unless yo uthink that some have erroneously depended on openvino at build time.